azure-pipelines.yml

```diff
@@ -206,7 +206,14 @@ stages:
             - jpa-mariadb
             - jpa-mssql
           name: jpa_db
-          postgres: true
+
+      - template: ci-templates/native-build-steps.yaml
+        parameters:
+          timeoutInMinutes: 25
+          modules:
+            - jpa-mysql
+          name: jpa_mysql
+          mysql: true
```

Comment:
I think I would have made it part of the job below, which has only 2 tasks right now (jpa and jpa-postgresql).

There wouldn't be any port conflict so I think we are relatively safe.